core/src/main/resources/META-INF/native-image/io.micronaut/micronaut-core/native-image.properties

```diff
@@ -28,4 +28,5 @@ Args = --initialize-at-run-time=io.micronaut.core.io.socket.SocketUtils \
         --initialize-at-build-time=io.micronaut.core.annotation.AnnotationValue \
         --initialize-at-build-time=io.micronaut.core.annotation.AnnotationValueResolver \
         --initialize-at-build-time=io.micronaut.core.reflect.ReflectionUtils \
-        --features=io.micronaut.core.io.service.ServiceLoaderFeature
+        --features=io.micronaut.core.io.service.ServiceLoaderFeature \
+        --report-unsupported-elements-at-runtime
```

Comment:
I believe `--report-unsupported-elements-at-runtime` is deprecated and shouldn't be used

Reply:
I have found no other workaround